util/src/lib.rs

```diff
@@ -16,6 +16,7 @@
 
 #![warn(missing_docs)]
 #![cfg_attr(feature="dev", feature(plugin))]
+#![cfg_attr(feature="dev", feature(asm))]
```

Comment:
The `dev` feature is for dev dependencies like clippy. Something else should be used here